src/style.css

```diff
@@ -0,0 +1,146 @@
+.html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  display: inline-flex;
+}
+
+.card__container {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+}
+
+.card__img {
+  width: 160px;
```

Comment:
Maybe adding almost the same names for classes is not the best idea? But anyways it's not critical so leave it if you want.

Reply:
Agree with @nikitaovch99